DeutschJozsaAlgorithm/DeutschJozsaAlgorithm.ipynb

```diff
@@ -369,31 +393,34 @@
    ]
   },
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
+   "execution_count": null,
    "metadata": {},
+   "outputs": [],
    "source": [
-    "## Part II. Deutsch-Jozsa Algorithm\n",
+    "## Part II. Bernstein–Vazirani Algorithm\n",
```

Comment:
This is really weird, but I just realized that the task order in the notebook is the opposite from the task order in the project version, and it's been this way since day one. So the changes in the notebook are going to be minimal: DJ remains in part II and BV in part III, we just need to change the test numbers in `%kata` commands to match the renumerated tests. Given that all cells became code cells, it might be easiest to revert the change to the notebook altogether and do the small changes to the old version.